pkg/cvo/cvo.go

```diff
@@ -96,32 +113,48 @@ func New(
 	eventBroadcaster.StartRecordingToSink(&coreclientsetv1.EventSinkImpl{Interface: kubeClient.CoreV1().Events("")})
 
 	optr := &Operator{
-		nodename:      nodename,
-		namespace:     namespace,
-		name:          name,
-		releaseImage:  releaseImage,
+		nodename:     nodename,
+		namespace:    namespace,
+		name:         name,
+		releaseImage: releaseImage,
+		payloadDir:   overridePayloadDir,
+
+		minimumUpdateCheckInterval: minimumInterval,
+
```

Comment:
nit: remove this space both `minimumUpdateCheckInterval` and `defaultUpstreamServer` are related to `syncAvailableUpdates`

Reply:
Ok, will tackle in a follow up where I add more progress metrics and explicit error states